monkey/monkey_island/cc/next_ui/src/constants/events.constants.ts

```diff
@@ -0,0 +1,3 @@
+export enum Events {
+    LOGOUT = 'LOGOUT'
```

Comment:
Why do we need to fire events directly? can't we use dispatch?

Reply:
I don't know how we'd use a dispatch to trigger a redirect from a layout. This was the best way without any extra dependencies and little code

Reply:
If you think we can do it, try to make a commit